chain/chain/src/chain.rs

```diff
@@ -2950,18 +2948,12 @@ impl Chain {
     ) -> Result<Vec<bool>, Error> {
         let epoch_id = self.epoch_manager.get_epoch_id_from_prev_block(prev_block_header.hash())?;
         let protocol_version = self.epoch_manager.get_epoch_protocol_version(&epoch_id)?;
-        let relaxed_chunk_validation =
-            checked_feature!("stable", RelaxedChunkValidation, protocol_version);
-
-        if !relaxed_chunk_validation {
-            if !validate_transactions_order(chunk.transactions()) {
```

Comment:
This condition should remain right? The check above is `!relaxed_chunk_validation`

Reply:
We want to keep code where `relaxed_chunk_validation == true`, no?  Or am I misunderstanding something?